cdc/scheduler/internal/base/owner_scheduler.go

```diff
@@ -216,6 +216,11 @@ func (s *SchedulerV2) Close(ctx context.Context) {
 		zap.String("namespace", s.changeFeedID.Namespace),
 		zap.String("changefeedID", s.changeFeedID.ID))
 	s.deregisterPeerMessageHandlers(ctx)
+	s.ScheduleDispatcher = nil
+	s.messageRouter = nil
+	s.messageServer = nil
+	s.stats = nil
+	s.ownerRevision = 0
```

Comment:
Why do we need these? Since we are closing the scheduler, its status should no longer matter now.

Reply:
> Why do we need these? Since we are closing the scheduler, its status should no longer matter now.

These five clearing operation is unnecessary. I'll remove later.